pkgs/applications/terminal-emulators/wezterm/default.nix

```diff
@@ -69,8 +71,15 @@ rustPlatform.buildRustPackage {
   installPhase = ''
     for artifact in wezterm wezterm-gui wezterm-mux-server strip-ansi-escapes; do
       patchelf --set-rpath "${lib.makeLibraryPath runtimeDeps}" $releaseDir/$artifact
-      install -D $releaseDir/$artifact -t $out/bin
+      install -Dsm755 $releaseDir/$artifact $out/bin/$artifact
     done
+
+    install -Dm644 assets/shell-integration/* -t $out/etc/profile.d
```

Comment:
This shell integration is optional to use wezterm, if you put it in `$out/etc/profile.d` does it make it _always_ loaded ?